patroni/dcs/__init__.py

```diff
@@ -269,6 +270,18 @@ def set_ttl(self, ttl):
     def set_retry_timeout(self, retry_timeout):
         """Set the new value for retry_timeout"""
 
+    def set_loop_wait(self, loop_wait):
```

Comment:
The set_loop_wait is not used outside of the file it's defined, so it should be prefixed with an underscore.